yaml/thz504.yaml

```diff
@@ -5,3 +5,17 @@ esphome:
 
 packages:
   BASE: !include { file: wp_base.yaml}
+
+  ABLUFTTAUPUNKT:                 !include { file: wp_temperature.yaml, vars: { property: "ABLUFTTAUPUNKT"   }}
```

Comment:
This value is now duplicated in THZ504.yaml and base.yaml
I doubt that this compiles ... ABLUFTTAUPUNKT is not defined ... only ABLUFT_TAUPUNKT